CHANGELOG.md

```diff
@@ -67,10 +67,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Removed
 - Removed deprecated Traffic Ops Go Client methods.
-- Configuration generation logic in the TO API (v1) for:
-  - `ip_allow.config`
-  - `parent.config`
-  - `remap.config`
+- Configuration generation logic in the TO API (v1) for all files and the "meta" route.
```

Comment:
can you elaborate on this a bit? i think it's important to make very clear that ORT version whatever that uses these TO api endpoints can no longer be used. instead ORT version whatever that leverages atstccfg needs to be used in conjunction with the current version of TO. I would hate for somebody to upgrade TO to 5.0 and wonder why all their caches are now getting a 410 rather than the ats config files.